.github/workflows/ci.yaml

```diff
@@ -90,7 +90,8 @@ jobs:
       with:
         firefox-version: latest
     - name: Download geckodriver
-      run: curl -sL $(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest | grep  "https.*linux64.tar.gz" | head -n 1 | awk '{print $2}' | sed 's|[\"\,]*||g') | tar xzvf -
+      # id: 97834079 -> "v0.33.0"
+      run: curl -sL $(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/97834079 | grep  "https.*linux64.tar.gz" | head -n 1 | awk '{print $2}' | sed 's|[\"\,]*||g') | tar xzvf -
```

Comment:
Can we use `https://github.com/mozilla/geckodriver/releases/download/v0.33.0/geckodriver-v0.33.0-linux64.tar.gz` directly?

I can use an environment variable to hold the version.